src/Microsoft.CodeAnalysis.Razor.Workspaces/ErrorReporter.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Razor
+{
+    internal abstract class ErrorReporter : IWorkspaceService
```

Comment:
I added an error reporting service so we can send errors somewhere when we encounter them in the background worker.